### ISSUE TITLE
Extension Gallery thumbnail submissions again

### ISSUE THREAD

**Issue by xTvii** [TurboWarp/extensions#1607]
#1079 but like the icons are _miles_ better . and i also provide reasoning towards my changes ( btw they get cropped automatically + im kinda lazy ..... and most cropping tools online i can find are making the whole file break so smh )

**' S-Grab '** im just resubmitting it because it didnt get noticed and i think s grab really needs a visual upgrade :?
![S-Grab](https://github.com/user-attachments/assets/62e3ffbb-0ac5-48b2-a28e-24eef7ff1efb)

**' Simple 3D '** it had text and im removing the text so translators dont have to deal with svg files
![Simple 3D](https://github.com/user-attachments/assets/124a521c-2884-4c6e-8849-79682de1b1f2)

**' Close Confirm '** we hate text but we LOOVEEE consistency !!!!! all assets in the old ( current ) image are either squares or strings of text :(
![Close Confirm](https://github.com/user-attachments/assets/ceb2eede-84cc-4c90-a5e1-493c0092621e)

**' Vibrate '** ngl something about the original design is just .. off . also the outlines were opaque
![Vibrate](https://github.com/user-attachments/assets/c3e43cd9-0d8f-41c5-87f1-10e09b80918c)

**' Time '** & **' Consoles '** same as simple 3d . i removed the text and added outlines for the main icons :O
![Consoles(1)](https://github.com/user-attachments/assets/6c1a52b3-e2e1-475a-8b8a-649a25019366)
![Time(1)](https://github.com/user-attachments/assets/3d3f0972-27d2-4385-8e50-d416bd220304)

**' Font Manager '** same as simple 3d &&& i changed some thing with the icon B)
![Font Manager](https://github.com/user-attachments/assets/3a1462b9-c472-4051-a9f8-cea734e5b59b)

**' Numerical Encoding '** i tried to change it in 1079 but it turned out absolutely abysmal . so i redid the redo . ngl it looks much cleaner
![Numerical Encoding2](https://github.com/user-attachments/assets/eab0bbd8-0fe1-4faf-b1e2-572a86039673)

**' HTTP '** i redid it .... woah !!! changed backround to something nicer & fixed the icon
![HTTP](https://github.com/user-attachments/assets/4edb63fe-12ed-42a9-9bf9-4013def06792)

**' WebSocket '** holy heck !!! what ! ? ! ? basically just http changes except i did some cool stuff see for yourself idk
![Websocket](https://github.com/user-attachments/assets/7b671e6d-4728-4723-9404-3e016c6691ca)

thats all my changes that ive made so farr & this was all made in the turbowarp editor . cool emoji w sunglasses . anyways im gonna tag you in this garbo youre not gonna stop me I WANNA SEE YOU TRY MWAWAHWAHWHAHH I WANNA SEE Y

**Comment by xTvii** [comment]
btw critisissisimemgmgm appreciated !

**Comment by veggiecan0419** [comment]
@xTvii What was the problem with Simple 3d before? Personally, I think it looked good how it was before 🤷‍♂️?

**Comment by xTvii** [comment]
it has text , which garbo said was redundant & hard to translate in #1518

**Comment by Xeltalliv** [comment]
Sorry, but I'm really not a fan a your new Simple3D image, and view it as a downgrade.

Firstly, the new image is less clear at a glance. The old one has a grid with perspective and letters `3D` which makes it clear it's something related to 3D at a glance, even if you don't know what `Simple` means and quickly scrolling through the list of extensions only looking at thumbnails. Your new image just looks like a confusing shape, and most wouldn't realize it represents 3 axis of 3D from the image alone.

> so translators dont have to deal with svg files

The original image is PNG genereated using extension itself. Changing text in it is easy. Generating higher resolution versions is also easy. The project can be found starting from opening extension in turbowarp, pressing "Open extra resources" > "Demos" > "Extension image".

Another thing is the theming of documentation site (https://xeltalliv.github.io/simple3d-extension/) is entirely based around the white-cyan grid from the currect thumbnail.

**Comment by LilyMakesThings** [comment]
@Xeltalliv text in extension images is discouraged anyway because it is superfluous to the very helpful extension title and description; it's an unnecessary detail that cannot be translated and looks out of place.

**Comment by David-Orangemoon** [comment]
> @Xeltalliv text in extension images is discouraged anyway because it is superfluous to the very helpful extension title and description; it's an unnecessary detail that cannot be translated and looks out of place.

But it would seem rather odd to remove it if it is core to the extension's identity right? As he said the thumbnail is made with the extension.

**Comment by namelessisbackbutnottodobadthingsiswear** [comment]
> > @Xeltalliv text in extension images is discouraged anyway because it is superfluous to the very helpful extension title and description; it's an unnecessary detail that cannot be translated and looks out of place.
> 
> But it would seem rather odd to remove it if it is core to the extension's identity right? As he said the thumbnail is made with the extension.

Perhaps simplify the text in the title to "3D?" "Simple 3D" would have to be translated since it's English but most languages seem to use the phrase "3D" to indicate the third dimension

**Comment by namelessisbackbutnottodobadthingsiswear** [comment]
> btw critisissisimemgmgm appreciated !

I like the new images (especially close confirm and s-grab). only one I don't really like is numerical encoding, I could be biased since I made the last one but the one you made doesn't really have anything to do with numerical encoding and (for lack of a better adjective) gives off a "so-retro" vibe

**Comment by SharkPool-SP** [comment]
I dont like Font manager. Also some of these were already done, just @GarboMuffin hasn't implemented them yet. If he reads this, don't change the font manager icon. I can do it myself as its my extension

**Comment by veggiecan0419** [comment]
> I dont like Font manager. Also some of these were already done, just @GarboMuffin hasn't implemented them yet. If he reads this, don't change the font manager icon. I can do it myself as its my extension

@SharkPool-SP Yes, but it is still absolutely up to @GarboMuffin. Just because its **your** extension doesn't mean it can't be changed without **your** permission. This is Garbo's repository, and **you** decided to contribute. Not saying I prefer the new one or the older one, but I absolutely disagree with the
 > I can do it myself as its my extension

part. If you want full control over **your** code, don't contribute to **other's** (open source)  projects.

**Comment by LilyMakesThings** [comment]
> > I can do it myself as its my extension
> 
> part. If you want full control over **your** code, don't contribute to **other's** (open source) projects.

I don't see what being open source has anything to do with this? He has just as much of a right as anyone else to be critical over the thumbnails of any extension, else this issue post wouldn't even exist. He volunteered to do it himself and he **is** the most qualified to do it, given that it is his extension. Your statement is completely irrelevant.

**Comment by veggiecan0419** [comment]
> > > I can do it myself as its my extension
> > 
> > 
> > part. If you want full control over **your** code, don't contribute to **other's** (open source) projects.
> 
> I don't see what being open source has anything to do with this? He has just as much of a right as anyone else to be critical over the thumbnails of any extension, else this issue post wouldn't even exist. He volunteered to do it himself and he **is** the most qualified to do it, given that it is his extension. Your statement is completely irrelevant.

@LilyMakesThings Yes, But how he said it is as if he is in charge of who can change the image when hes not. Why i said open source is because he knows that this repo is open to contributions by anyone, and that means there might be a change to something he made, which he might not like.

**Comment by veggiecan0419** [comment]
@LilyMakesThings, I don't want to start an argument here, But I didn't say SharkPool wasn't the best person to do it, nor did I say he can't do it. But I dont think its acceptable for him to basically say: "Garbo, don't use Xtvii's image. I can do it better, and its **my** extension". Garbo can decide if he wants to use Xtvii's one, and it doesn't nessecarily matter who's extension it is. Sure Garbo considers the extension creator, but that doesn't mean that he can't change it to whatever he likes.

**Comment by SharkPool-SP** [comment]
> @LilyMakesThings, I don't want to start an argument here, But I didn't say SharkPool wasn't the best person to do it, nor did I say he can't do it. But I dont think its acceptable for him to basically say: "Garbo, don't use Xtvii's image. I can do it better, and its **my** extension". Garbo can decide if he wants to use Xtvii's one, and it doesn't nessecarily matter who's extension it is. Sure Garbo considers the extension creator, but that doesn't mean that he can't change it to whatever he likes.

What's stopping Garbo from using this and then 3 days later another image. Seems like a waste of everyone's time

**Comment by GarboMuffin** [comment]
hi, will look at them when able, im just going to disable comments here since it seems unlikely for anything productive to result from more conversation

**Comment by GarboMuffin** [comment]
i will unlock this but please leave discussion to **the images themselves** and not commentary about who should be the person to make an image for an extension, which is unproductive. if you don't like it, explain what's actually wrong with the new image or make a better one

**Comment by xTvii** [comment]
I tried to make them good but i've clearly done _something_ wrong to stir up all this tacky conversation ... I'll try even better if I decide to do it again again !!

**Comment by veggiecan0419** [comment]
> @xTvii, no, don't worry; you did nothing wrong. Just another day in TurboWarp 🙄

True say...

**Comment by unknown07724** [comment]
@xTvii if you want to you could create a new banner for an extension in the works called positron, it has a banner but it looks like garbage, it doesn't even fit the thumbnail, keep the circle with plus in it as a positron has a electrical charge of +1 
and put it in the center, I will soon put the source code in it's issue, not a pull request because I don't know how those work

**Comment by unknown07724** [comment]
#1663 is positron's issue

**Comment by unknown07724** [comment]
> btw sorry if my simple3d sucked , its tradition to get one completely screwed up every time i try !!

my extension's banner is worse than yours

**Comment by xTvii** [comment]
👀

**Comment by GarboMuffin** [comment]
they can be posted in #90

**Comment by GarboMuffin** [comment]
a lot of time has passed here. i do not remember what came of those earlier conversations
